layouts/partials/meta_breadcrumb_json_ld.html

```diff
@@ -0,0 +1,28 @@
+{{- $breadcrumbScratch := newScratch -}}
+{{- $crumbScratch := newScratch -}}
+{{- $crumbScratch.Set "itemListElement" (slice) -}}
+{{- $breadPos := 0 -}}
+{{- range .Ancestors.Reverse -}}
```

Comment:
As you point out this requires Hugo v0.109.0. Can we add a check to see if `.Ancestors` is set and only then run this code. Maybe add a comment about why the check is in place as well.

Reply:
Will do.

Reply:
As per above, I've done this by documenting the need for v0.109.0 when using `breadcrumbSections`.